mlabs/src/Mlabs/NFT/Types.hs

```diff
@@ -536,25 +541,32 @@ data QueryResponse
   deriving anyclass (FromJSON, ToJSON)
 
 -- | Easy type to find and use Nodes by.
-data PointInfo = PointInfo
-  { pi'datum :: DatumNft
+data PointInfo a = PointInfo
+  { pi'datum :: a
```

Comment:
If we generalise the type (which I think is a great idea), let's also call the field `pi'data`.